lib/Conversion/FIRRTLToRTL/LowerToRTL.cpp

```diff
@@ -1857,9 +1856,11 @@ LogicalResult FIRRTLLowering::visitExpr(InvalidValuePrimOp op) {
 LogicalResult FIRRTLLowering::visitExpr(HeadPrimOp op) {
   auto input = getLoweredValue(op.input());
   if (!input)
-    return failure();
+    return handleZeroBit(op.input(), [&]() { return setLowering(op, Value()); });
```

Comment:
I don't think this is needed.  A zero bit input will necessarily result in a zero bit result, which is handled elsewhere.